web/db/migrations/20260729120000_ios_app_namespace_isolation/migration.sql

```
@@ -0,0 +1,28 @@
+
+DROP INDEX "iroh_endpoint_bindings_active_slot_unique";
+
+CREATE UNIQUE INDEX "iroh_endpoint_bindings_active_slot_unique"
+  ON "iroh_endpoint_bindings"
+    ("user_id", "client_namespace", "device_uuid", "tag")
+  WHERE "revoked_at" IS NULL;
+
+CREATE INDEX "device_tokens_user_bundle_idx"
+  ON "device_tokens" ("user_id", "bundle_id");
+
+DROP INDEX "device_tokens_device_token_unique";
```

SQLFluff (4.2.2)

[error] DROP statement should use CONCURRENTLY to avoid locking the table.

(PG01)